setup.cfg

```diff
@@ -85,15 +85,16 @@ runtime =
     %(base-runtime)s
     airspeed-ext>=0.6.3
     amazon_kclpy>=2.0.6,!=2.1.0
-    antlr4-python3-runtime==4.13.1
+    antlr4-python3-runtime>=4.13.1
```

Comment:
I would prefer having this pinned at a precise version for now, as it is preferable to have the runtime library version match the version that generated the parsers in the first place. If this is not true, warning may also be triggered, but I have also had to adjust the interpreter recently to match some changes in the latest version (4.13.1). Unfortunately, the parser generator library is run locally and needs to be manually installed; I recently tried to [formalise the build](https://github.com/localstack/localstack/pull/10193) to help resolve this gap. We'll probably need to discuss this further as another option may be to generate these grammars on install.

Reply:
But wouldn't the test cover this case? This change is just removing the exact pin in the `setup.cfg`.
The dependency is still pinned in the requirements files (see https://github.com/localstack/localstack/pull/10127).
This change basically allows updating the pin on a weekly basis (with an automated workflow creating a PR). On this PR all tests will be triggered, and we can see if the version changes are breaking anything. If not, we'll merge the new pins. If it does, we would try to address them.

But that's your call. If you explicitly want to pin the version, I can just re-add the exact pin.

Reply:
I believe this is a fantastic idea, as it would not only keep SFN up to date with the latest ANTLR4 updates but also streamline our process. For optimal integration with SFN, it would be best if the generator library were updated simultaneously, allowing parsers to be automatically generated upon installation. If we only update the runtime library, we would have to manually regenerate the parsers to address at least the programmatic warnings that will appear. 
Therefore, I propose we enable automatic updates once we have automated the generator workflow within the installation process, ensuring both the generator and parsers are automatically updated.

Reply:
Perfect, then I will re-introduce the pin, and you can remove it once you automated the generation :)

Reply:
Thank you, I hope to see this soon!!